terragrunt/aws/cloudfront/cloudfront.tf

```diff
@@ -69,9 +68,8 @@ resource "aws_cloudfront_response_headers_policy" "simplify_privacy_app_headers_
       override = true
     }
     content_security_policy {
-      content_security_policy = "script-src 'self' 'unsafe-inline' cdnjs.cloudflare.com https://www.googletagmanager.com https://www.google-analytics.com https://${var.domain}/dist/js/ https://edbi6zcop5ta2t5wb6hb5y6kja0rsajp.lambda-url.ca-central-1.on.aws/dist/js/questions-1.3e7d142ac08d78a0424a.js; default-src 'self'; base-uri 'none'; font-src 'self' https://fonts.gstatic.com/; img-src 'self' data: https://www.google-analytics.com; style-src 'self' https://fonts.googleapis.com/ 'unsafe-inline';"
-
-      override = false
+      content_security_policy = "script-src 'self' 'unsafe-inline' 'unsafe-eval' cdnjs.cloudflare.com https://www.googletagmanager.com https://www.google-analytics.com https://${var.domain}/dist/js/ https://${var.generated_statement_lambda_function_url}.cell-1-lambda-url.ca-central-1.on.aws/dist/js/questions-1.3e7d142ac08d78a0424a.js; default-src 'self'; base-uri 'none'; font-src 'self' https://fonts.gstatic.com/; img-src 'self' data: https://www.google-analytics.com; style-src 'self' https://fonts.googleapis.com/ 'unsafe-inline';"
```

Comment:
I checked the CSP that's being created and I think you'll need to massage the `var.generated_statement_lambda_function_url` since it's coming out like this in the CSP:
```
https://https://edbi6zcop5ta2t5wb6hb5y6kja0rsajp.lambda-url.ca-central-1.on.aws/.cell-1-lambda-url.ca-central-1.on.aws
```